milestone2.ts

```diff
@@ -93,23 +101,39 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+
+let findIndexAns: GetNumber = (data) => {
+  for (let i=0; i<data.length; i++) {
+    if (data[i].city === "SF") {
+      return i;
+    }
+  }
+  return -1;
```

Comment:
Please do this question without a for loop! remember the data handling functions we talked about during [Bootcamp Session 4](https://h4i.notion.site/Typescript-NPM-b02ee93561844b889a072541d65044d4). there's a functino that will return the first index of a condition